### DISCUSSION TITLE
Unable to see tiles/connect to Jetblack Victory

### DISCUSSION THREAD

**Discussion by lazpujilaksono** [cagnulein/qdomyos-zwift#2926] [Q&A]
Hi All,

I'm having issues with connecting to my Jetblack Victory. The QZ app finds 'Victory' via bluetooth but the tiles doesn't show up which I assume is because it is not connected to any device.

I have attached some screenshoots below to show that I have enabled bluetooth access to the app, and trying to connect Victory.

I look forward to your help.

![IMG_0207](https://github.com/user-attachments/assets/03f9c11a-ce1e-4827-a0af-fd70a4362278)
![IMG_0206](https://github.com/user-attachments/assets/5d28f251-a840-476a-bbdf-a6edb7b4bb78)
![IMG_0205](https://github.com/user-attachments/assets/f70dc6a0-afac-4adc-88c3-dd895b2d0a7a)

**Comment by cagnulein** [comment]
Hi, set it under ftms bike setting under the bike options in qz 
Let me know

**Reply by lazpujilaksono** [reply]
Hi, Thanks. It is working now. I am also able to change the virtual gears with Zwift click in the QZ app. But when moving to MyWhoosh, it doesn't work well. As I periodically press the Zwift Click, there is seemingly no change in resistance, until it suddenly change into a very hard 'gear'. Do you know what is going on? Thanks

**Reply by cagnulein** [reply]
hi @lazpujilaksono i suggest to try the last beta. 
send me an email to roberto.viola83 at gmail.com indicating this ticket and if you are on android or iOS. Thanks
